webapp/lib/app/nook/view.dart

```diff
@@ -945,9 +960,9 @@ class ConversationListPanelView {
     for (var conversation in conversations) {
       ConversationSummary summary = _phoneToConversations[conversation.docId];
       if (summary == null) {
-        summary = new ConversationSummary(conversation.docId, "", false);
+        summary = new ConversationSummary(conversation.docId, "", conversation.messages.last.datetime, false);
```

Comment:
this needs to include the use case for when there are no messages in the conversation, so something like this: 
```suggestion
        summary = new ConversationSummary(conversation.docId, "", conversation.messages.isEmpty ? null : conversation.messages.last.datetime, false);
```

and can you also just double check if the `ConversationSummary` works ok with a `null` datetime?